src/js/control-bar/play-toggle.js

```diff
@@ -58,6 +58,26 @@ class PlayToggle extends Button {
   }
 
   /**
+   * This gets called once after the video has ended and the user seeks so that
+   * we can change the replay button back to a play button.
+   *
+   * @param {EventTarget~Event} [event]
+   *        The event that caused this function to run.
+   *
+   * @listens Player#seeked
+   */
+  handleSeeked(event) {
+    // remove the ended class
+    this.removeClass('vjs-ended');
+
+    if (this.player_.paused()) {
```

Comment:
why would we not be paused when we're ended?

Reply:
When you click the button, it will cause a play and then a seek, which would cause the play button to show, even though the video is playing.

Reply:
Ah, I see.